Orchestrator.py

```diff
@@ -68,6 +70,12 @@ def __init__(self, numRobots, initX, initY):
                     'targetCell':         None,
                     # [(cx1, cy1) -> (cx2, cy2) -> .... -> (cxTarget, cyTarget)]
                     'currentPath':        [],
+                    # heartbeat of last estimated PDR from DotBot
+                    'heartbeat':          1,
```

Comment:
if "heartbeat" is some sort of PDR the DotBot measured, ABSOLUTELY change its name

Reply:
we agreed on the naming previously when we first discussed this in meetings. will rename to estimatedPDR